README.md

```diff
@@ -47,14 +47,9 @@ helm install "${SKYWALKING_RELEASE_NAME}" \
   --set ui.image.tag=9.2.0
 ```
 
-## Install released version using Apache Jfrog Helm repository (<= 4.3.0)
```

Comment:
Why do you change the order? I think release version should come first.

Reply:
ok, I'll change it back.